pandas/tests/series/test_arithmetic.py

```diff
@@ -82,3 +82,109 @@ def test_pow_ops_object():
     expected = Series(b.values ** a.values, dtype=object)
 
     tm.assert_series_equal(result, expected)
+
+
+# ------------------------------------------------------------------
+# Unary
+
```

Comment:
I think @jbrockmendel move things like this to pandas/tests/test_arithmetic.py ?

Reply:
unary ops we haven't been collecting in tests/arithmetic

Reply:
Moved some tests to `test_arithmetic`. Is there any guide between `test_arithmetic` and `test_operators`?

Reply:
It hasn't come up enough for a policy to develop.  The first thing I'd check is internal consistency: if they are in test_foo for DataFrame, they should also be in test_foo for Series and Index.

Also, if this is behavior where we expect the behaviors of `op(index)`, `op(Series(index))` and `op(DataFrame(index))` to be analogous, they could go in something like tests/arithmetic/test_unary_ops.py

Reply:
OK. Currently, similar tests are written in either `test_arithmetic.py` or `test_operators.py`. Will submit an issue to clarify this.